.travis.yml

```diff
@@ -3,12 +3,6 @@ node_js:
 - 6
 - 4
 
-matrix:
-  fast_finish: true
```

Comment:
`fast_finish` is only applied when there is a build with allowed failures set up. We don't need to keep it if we remove OSX builds (but unfortunately, a build that times out after an hour will not have failed yet, so `fast_finish` doesn't apply in these cases as well...)